test/fixtures/output/billing/list.golden

```diff
@@ -0,0 +1,5 @@
+      Start Date      |      End Date       | Granularity |  Line Type  |     Product     | Resource ID | Resource Display Name | Environment ID | Network Access Type |     Price     | Original Amount | Discount Amount | Amount  
+----------------------+---------------------+-------------+-------------+-----------------+-------------+-----------------------+----------------+---------------------+---------------+-----------------+-----------------+---------
+  2023-01-01 00:00:00 | 2023-01-02 00:00:00 | DAILY       | LINE_TYPE_1 | KAFKA           | lkc-0       | kafka_1               | env-123        | INTERNET            | $0.123 USD/GB | $70.00          | $20.00          | $50.00  
```

Comment:
Do we need to print the start time here too? It looks like it's always 00:00:00... will that ever change?

Reply:
yes, so if we query for multiple days of costs, it will aggregate based on the granularity (currently only daily). So one row will have `2023-01-01 00:00:00 - 2023-01-02 00:00:00` and the next row could have `2023-01-02 00:00:00 - 2023-01-03 00:00:00`

Reply:
So granularity is going to be able to be changed in the future? Can we remove the timestamp if the granularity is a multiple of a day?

Reply:
yeah i think that's a good idea